src/main/java/calculate/domain/Executor.java

```diff
@@ -0,0 +1,43 @@
+package calculate.domain;
+
+public class Executor {
+
+    private Expression expression;
+
+    public Executor(String input) {
+        this(new Expression(input));
+    }
+
+    public Executor(Expression expression) {
+        this.expression = expression;
+    }
+
+    public int calculate() {
+        String[] inputList = expression.split();
+        int result = Integer.parseInt(inputList[0]);
+
+        for (int i = 1; i < inputList.length; i += 2) {
+            String operator = inputList[i];
+            int value = Integer.parseInt(inputList[i+1]);
+
+            if (operator.equals("+")) {
+                result = Calculator.add(result, value);
+            }
```

Comment:
```suggestion
    PLUS("+") {
        @Override
        public int calculate(int operand1, int operand2) {
            return operand1 + operand2;
        }
    }
```
사칙 연산이 서로 연관되어 있는 상수이니 enum으로 구현해 보면 어떨까?
각 enum에 해당하는 계산 로직을 위와 같이 구현한 후 반복해서 발생하고 있는 if문을 제거해 보는 시도를 해보면 어떨까?
힌트: enum 또한 class와 완벽히 같은 방식으로 인터페이스, 상속 등의 구현이 가능함